transport/src/main/java/io/camunda/zeebe/transport/stream/impl/ClientStreamService.java

```diff
@@ -29,13 +33,41 @@ public ClientStreamService(final ClusterCommunicationService communicationServic
         new ClientStreamManager<>(
             new ClientStreamRegistry<>(),
             new ClientStreamRequestManager<>(communicationService, actor));
+    this.communicationService = communicationService;
+  }
+
+  @Override
+  protected void onActorStarted() {
+    // TODO: Define an PushResponse to inform server if push was successful or not. Currently, an
```

Comment:
The exception is propagated back to the broker. But if we want to provide actionable error codes, then it would make sense to define a proper error response. I think it is also cleaner to do it, compared to sending the whole exception stack trace. But it is not critical. The current state achieves what we want to do - error is received by the broker, will be propagated back to the engine.